proto/src/determined/trial/v1/trial.proto

```diff
@@ -87,6 +87,29 @@ message Trial {
   CheckpointWorkload best_checkpoint = 10;
 }
 
+// TrialProfilerMetricSummary is a summary of a single metric.
+message TrialProfilerMetricSummary {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "float", "labels" ] }
+  };
+  // The average of the metric, over the last 10 minutes of its collection.
+  float average = 1;
+  // A bool describing if the metric can be considered stable or not.
+  bool stable = 2;
+  // The labels for this series.
+  TrialProfilerMetricLabels labels = 3;
+}
+
+// TrialProfilerSummary is a summary of the information gather about the trial
+// during profiling.
+message TrialProfilerSummary {
+  option (grpc.gateway.protoc_gen_swagger.options.openapiv2_schema) = {
+    json_schema: { required: [ "trial_id", "name" ] }
```

Comment:
I don't understand this line. Where are these fields required to exist in the summary?

Reply:
this is just old, i'll remove

Reply:
```suggestion
    json_schema: { required: [ "metric_summaries" ] }
```